dev/conductor/core/test/repository_test.dart

```diff
@@ -497,9 +502,113 @@ enabled_branches:
 
       final FrameworkRepository framework = FrameworkRepository(checkouts);
       expect(
-        () async => (await framework.ciYaml).enableBranch('master'),
-        throwsExceptionWith('.ci.yaml already contains the branch master'),
+        () async => (await framework.ciYaml).enableBranch(FrameworkRepository.defaultBranch),
+        throwsExceptionWith('.ci.yaml already contains the branch ${FrameworkRepository.defaultBranch}'),
+      );
+    });
+
+    test('framework repo set as localUpstream ensures requiredLocalBranches exist locally', () async {
+      const String commit = 'deadbeef';
+      const String candidateBranch = 'flutter-1.2-candidate.3';
+      bool createdCandidateBranch = false;
+      processManager.addCommands(<FakeCommand>[
+        FakeCommand(command: <String>[
+          'git',
+          'clone',
+          '--origin',
+          'upstream',
+          '--',
+          FrameworkRepository.defaultUpstream,
+          fileSystem.path.join(rootDir, 'flutter_conductor_checkouts', 'framework'),
+        ]),
+        FakeCommand(
+          command: const <String>['git', 'checkout', candidateBranch, '--'],
+          onRun: () => createdCandidateBranch = true,
+        ),
+        const FakeCommand(
```

Comment:
Would it be possible to hit this [for loop](https://github.com/christopherfujino/flutter/blob/50448f06528c6d3747ae8e5bcc17137f81d51dd2/dev/conductor/core/lib/src/repository.dart#L172) by calling `lazilyInitialize` instead of calling them manually?

Reply:
The call on line 562 is calling `get checkoutDirectory` which calls `lazilyInitialize`, they're not being called manually

Reply:
Ah ok, the fake commands verifies if the checkout works on all the branches.